### ISSUE TITLE
Background color issue in Theme Data

### ISSUE THREAD

**Issue by noorj-in** [lrferreiro/flutter_swipe_button#6]
Hello @lrferreiro,

I hope you are doing well. I have a concern related to this package. I'm using the Flutter SDK 3.22.0 version and in this version some drastic changes have occurred. In our project, we are using your package and we are facing an issue with the **theme.backgroundColor** in the swipe_button.dart. Can you please update it to **theme.colorScheme.secondary** so that it will work for us? Can you please fix this and post it as soon as possible?

**Comment by IndigoSoftwares21** [comment]
Hello Bro
https://github.com/IndigoSoftwares21/flutter_swipe_button/
here you go

```yaml
  flutter_swipe_button:
    git:
      url: https://github.com/IndigoSoftwares21/flutter_swipe_button.git
```

**Comment by noorj-in** [comment]
Thank you for the prompt response, Bro.

**Comment by lrferreiro** [comment]
Fixed in version ^2.1.3
